### DISCUSSION TITLE
hands too long

### DISCUSSION THREAD

**Discussion by Treawolfe** [lllyasviel/Fooocus#2077] [General]
Hay, I just wanted to ask if you also have problems with your hands being too long? I have tried different Loras but none seem to solve the problem, even later inpainting doesn't change anything. The hands are just unnaturally elongated even with lots of negative promts like bad hands and so on

**Comment by poor7** [comment]
What settings and promts do you use? Show what result you get.

**Reply by Treawolfe** [reply]
I can't insert pictures here, there is always an error but I'll try to describe it. The carpus, i.e. the area between the arm and metacarpal, is too long in my results. It's just a little bit but is immediately noticeable. My prompts are normal, e.g. "woman with long blonde hair is doing a photo shoot on a bridge with skyline in the background, perfect hands, skin pores, detailed fingers" and other similar inputs. Negative promts would be, for example, "Bad Hands, ugly, deformities, deformed hands,"

**Reply by poor7** [reply]
Example: Fooocus 2.1.864, Seed:1234, without Negative prompt, without Styles, Promt: side view woman holding a camera to her face and taking a photo on a bridge, long blonde hair, background:skyline
![image (2)](https://github.com/lllyasviel/Fooocus/assets/154618384/823916bc-d27e-40ea-9426-d3d48c9c789e)
